src/app.js

```diff
@@ -282,22 +285,70 @@ go.app = function() {
 
         self.states.add('state_suburb', function(name) {
             return new LocationState(name, {
+                map_provider: new OpenStreetMap({
```

Comment:
There's some hardcoded non-dry code here that will have to change in any case with the next release of `js-box-location` that should include landing of `https://github.com/praekelt/go-jsbox-location/issues/27`.